README.md

```diff
@@ -59,6 +63,15 @@ func main() {
 
 	c.SubscribeTopics([]string{"myTopic", "^aRegex.*[Tt]opic"}, nil)
 
+	sig := make(chan os.Signal, 1)
+	signal.Notify(sig, syscall.SIGINT)
+	go func() {
+		<-sig
+		fmt.Println("Keyboard interrupted.")
+		c.Close()
+		os.Exit(1)
```

Comment:
Minor: If we use a `closed` boolean as given in the other comment, we won't need to use os.Exit at all